### ISSUE TITLE
Update "Known bugs" section of https://joplinapp.org/ to include orphaned resource issue

### ISSUE THREAD

**Issue by ioogithub** [laurent22/joplin#4035]
Users should be aware that once a resource is uploaded to a remote sync target that it can't be deleted even though the app interface would make it appear that the change was synchronized. This leads the user to believe that the resource has been deleted. This limitation is a potential data security issue.

Perhaps a warning or disclaimer should be placed on https://joplinapp.org/ under the "Known bugs" section.

Also, a link to instructions on how to properly deal with the issue would be helpful.

Issue: https://github.com/laurent22/joplin/issues/932

**Comment by laurent22** [comment]
That's not the bug as far as I know. The bug only affects resources that have been attached, and then deleted before they were synced.

Also if you are concerned about security, the best option is to use E2EE. If you upload data to third party servers without encryption, that data could be fetched by someone else and then it doesn't matter if you delete it or not.

**Comment by ioogithub** [comment]
Okay so I opened a bug report detailing how I deleted several resources from a note. 

https://github.com/laurent22/joplin/issues/4030

but the bug tag was removed. I can reproduce the behavior. Even after a successful sync  and even though the resources have been removed locally they still exist on the remote server so they would be orphaned.

I agree with the E2EE issue however to the user it appears that the sync was successful and the resources were removed however the data still exists on the remote server.

I can reproduce this issue. Why was the bug tag removed without any discussion?
